internal/pkg/skuba/deployments/ssh/systemd.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package ssh
+
+import (
+	"k8s.io/klog"
+)
+
+// IsServiceEnabled returns if a service is enabled
+func (t *Target) IsServiceEnabled(serviceName string) (bool, error) {
+	klog.V(1).Info("checking if skuba-update.timer is enabled")
+	if stdout, _, err := t.silentSsh("systemctl", "is-enabled", serviceName); stdout == "enabled" {
```

Comment:
we still have an issue here... `silentSsh` will populate `err` if the service is disabled, because of the exit-code (1).

we either have to disregard the exit code in this case, or we find another command that tells us the same thing but with a (0) exit code

Reply:
we could do sth like:

```
t.silentSsh("systemctl", "is-enabled", serviceName, "||", ":")
```

but then we will never catch or return an error

Reply:
addressed in a separate commit. feel free to comment on this or suggest an alternative